galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/resources/CPSFacade.java

```diff
@@ -50,7 +50,7 @@ public CPSNamespace getNamespace(String name) throws ConfigurationPropertyStoreE
         getNamespaces();
         CPSNamespace namespace = bakedInNamespaceMap.get(name);
         if (namespace == null) {
-            new CPSNamespace(name, Visibility.NORMAL ,this.framework);       
+            namespace = new CPSNamespace(name, Visibility.NORMAL ,this.framework);       
```

Comment:
lol. Surprised this wasn't picked up in a unit test originally.
Obvious now I see you change it,.